index.php

```diff
@@ -1,3 +1,20 @@
+<?php
+
+require 'functions.php';
+
+$db = displayPDO();
+
+    $stmt = $db->query("SELECT `title` ,`intro_text` FROM `intro`");
+    $title_array = $stmt->fetch();
+    $stmt = $db->query("SELECT `paragraph_1`, `image`, `paragraph_2` FROM `about_me`");
+    $about_me_array = $stmt->fetch();
+    $stmt = $db->query("SELECT `title`, `image`, `paragraph` FROM `projects`");
+    $projects_array = $stmt->fetch();
+
+$data = array_merge($title_array, $about_me_array, $projects_array);
```

Comment:
this line doesnt do anything, you are merging the arrays then using the seperate ones instead of the data one. just delete this line

Reply:
Done.